content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,147 @@
+title: Empower your documentation with addons
+date: 2024-04-18
+description: Read the Docs Addons let documentation authors add features to their documentation such as analytics, version feedback, and search as you type to documentation in a modular way where authors have control.
```

Comment:
What is `version feedback`?

Reply:
I'm not 100% sure. David wrote this description; but it could be either:

1. pull request, non-stable and latest notifications shown on top of the page
2. doc diff

We can make it more specific if we want like "version notifications" or "version differences"